aws/raw-to-ami.sh

```diff
@@ -44,13 +48,13 @@ set_bucket_policies() {
 			"Effect":"Allow",
 			"Principal":{ "Service":"vmie.amazonaws.com" },
 			"Action": "sts:AssumeRole",
-			"Condition":{"StringEquals":{"sts:Externalid":"vmimport"}}
+			"Condition":{"StringEquals":{"sts:Externalid":"vmimport-${BUCKET_NAME}"}}
```

Comment:
The policy name must be `vmimport` - https://docs.aws.amazon.com/vm-import/latest/userguide/required-permissions.html. Relevant text:

***
VM Import/Export requires a role to perform certain operations on your behalf. You must create a service role named vmimport with a trust relationship policy document that allows VM Import/Export to assume the role, and you must attach an IAM policy to the role. For more information, see [IAM Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/roles-toplevel.html) in the IAM User Guide.
***

Reply:
thanks, removed